server-side-rendering/src/routes/+layout.svelte

```diff
@@ -0,0 +1,87 @@
+<script lang="ts">
+	import '@appwrite.io/pink';
+	import '@appwrite.io/pink-icons';
+	import AppwriteLogoDark from '$lib/images/appwrite-logo-dark.svg';
+	import LoginImageDark from '$lib/images/login-dark-mode.png';
+</script>
+
+<svelte:head><title>Server side rendering with Appwrite</title></svelte:head>
```

Comment:
Lets double check with Vincent, Steven, Denis, Laura or Sara to be 100% confident if we do `Server side rendering`, or some combination with dashes or different uppercase letters.

Reply:
It's like this